webapp/locales/de.json

```diff
@@ -102,7 +102,7 @@
       "projects": "Projekte",
       "invites": "Einladungen",
       "follows": "Folgen",
-      "shouts": "Shouts"
+      "shouts": "Empfehlungen"
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-07T22:37:07Z" title="Saturday, June 8th 2019, 12:37:07 am +02:00">Jun 8, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -102,7 +102,7 @@
       "projects": "Projekte",
       "invites": "Einladungen",
       "follows": "Folgen",
-      "shouts": "Shouts"
+      "shouts": "Empfehlungen"
```


Actually, I think I :heart: it. Indeed, shouting might have a negative connotation in English.

Shouting in Human Connection means to increase the reach of the post. Let more people see it. So I kind of like your translation of it! :+1:

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-06-14T00:03:33Z" title="Friday, June 14th 2019, 2:03:33 am +02:00">Jun 14, 2019</time>_

---


yes, we have a lot of expressions in English like `to shout something from the rooftop`, `to give a shout-out`, etc..

![Screenshot at 2019-06-13 21:02:41](https://user-images.githubusercontent.com/26943915/59474819-af234c80-8e1e-11e9-8913-5c29979f6eff.png)
![Screenshot at 2019-06-13 21:02:29](https://user-images.githubusercontent.com/26943915/59474825-b2b6d380-8e1e-11e9-8463-5d9fdb3e6e2c.png)